src/lib.rs

```diff
@@ -439,11 +446,12 @@ impl Compiler {
 
     /// Parses and typechecks `expr_str` followed by extracting metadata from the created
     /// expression
-    pub fn extract_metadata(&mut self,
-                            vm: &Thread,
-                            file: &str,
-                            expr_str: &str)
-                            -> Result<(ast::SpannedExpr<ast::TcIdent<Symbol>>, TcType, Metadata)> {
+    pub fn extract_metadata
+        (&mut self,
+         vm: &Thread,
+         file: &str,
+         expr_str: &str)
+         -> Result<(ast::SpannedExpr<ast::TcIdent<Symbol>>, TcType, Metadata)> {
```

Comment:
(It seems to have been the default for some time as well https://github.com/rust-lang-nursery/rustfmt/blame/fe69e975e3032d5aa6f73ecdf9e35288d09e4dc1/src/config.rs#L362)